### PR TITLE
ADD NoviaMC

### PR DESCRIPTION
**Pull request by sudo-Tiger**
## Type of change

- [ ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by sudo-Tiger** [comment]
Server is Online

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/noviamc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "server_name": "noviamc",
+  "nice_name": "NoviaMC",
+  "direct_ip": "NoviaMC.de",
+  "server_wildcards": [
+    "%.noviamc.de"
```
[the remark this answers is not in the document]
Comment (sudo-Tiger) [reply minecraft_servers/noviamc/manifest.json]: I removed the %

[the remark this answers is not in the document]
Comment (DoJapHD) [reply minecraft_servers/noviamc/manifest.json]: That's also not a wildcard...
Just remove it as a whole

[the remark this answers is not in the document]
Comment (sudo-Tiger) [reply minecraft_servers/noviamc/manifest.json]: Removed

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**